roles/hosted_engine_setup/tasks/restore_backup.yml

```diff
@@ -13,7 +13,28 @@
     --file=/root/engine_backup --provision-all-databases --restore-permissions
   environment: "{{ he_cmd_lang }}"
   register: engine_restore_out
+  ignore_errors: true
   changed_when: true
+- name: Pause the execution to let the user interactively handle restore failures
+  block:
+    - name: Let the user connect to the bootstrap engine to manually handle restore failures
```

Comment:
Let's be consistent and use `bootstrap engine VM`

Reply:
Done. We are not that consistent elsewhere, though :-(.